scripts/userstyles.yml

```diff
@@ -560,7 +575,7 @@ userstyles:
     icon: musicbrainz
     color: peach
     readme:
-      app-link: 'https://listenbrainz.org'
+      app-link: "https://listenbrainz.org"
```

Comment:
Let's avoid unrelated formatted changes (this doesn't look like Prettier anyway?).

Reply:
The section visible here on GitHub isn't everything, there are some app-links that are weirdly formatted as well.